version

```diff
@@ -1 +1 @@
-1.29.0
+1.29.1
```

Comment:
Isn't version just for the update script? Someone said that is saying 1.29.1 has been released.